forge-game/src/main/java/forge/game/staticability/StaticAbilityAlternativeCost.java

```diff
@@ -0,0 +1,73 @@
+package forge.game.staticability;
+
+import java.util.List;
+
+import com.google.common.collect.Lists;
+
+import forge.card.mana.ManaCostParser;
+import forge.game.Game;
+import forge.game.card.Card;
+import forge.game.cost.Cost;
+import forge.game.player.Player;
+import forge.game.spellability.SpellAbility;
+import forge.game.zone.ZoneType;
+
+public class StaticAbilityAlternativeCost {
+
+    static String MODE = "AlternativeCost";
+
+    public static List<SpellAbility> alternativeCosts(final SpellAbility sa, final Card source, final Player pl) {
+        List<SpellAbility> result = Lists.newArrayList();
+        final Game game = sa.getHostCard().getGame();
+        for (final Card ca : game.getCardsIn(ZoneType.STATIC_ABILITIES_SOURCE_ZONES)) {
+            for (final StaticAbility stAb : ca.getStaticAbilities()) {
+                if (!stAb.checkConditions(MODE)) {
+                    continue;
+                }
+
+                if (!apply(stAb, sa, source, pl)) {
+                    continue;
+                }
+
+                Cost cost = new Cost(stAb.getParam("Cost"), sa.isAbility());
+                // set the cost to this directly to bypass non mana cost
+                final SpellAbility newSA = sa.copyWithManaCostReplaced(pl, cost);
```

Comment:
copyWithManaCostReplaced vs copyWithDefinedCost

this might be for other cards that aren't updated to have AdditionalCost out of their ManaCost

or what do you think? @tool4ever

Reply:
This might be refactored in the Future too, Technically each Spell should have their Cost be defined by their Mana Cost, and the other parts should be defined by AdditionalCost StaticAbility

Reply:
I think for now it might need some logic to support both cases?

Like this I can no longer equip _Murderer's Axe_ without discarding via _Bruenor Battlehammer_ ;/

Reply:
ugh okay, i need to check this out

Bruenor Battlehammer should set cost to 0
New Perspectives should set the Cost of still Discard<Cardname>, but display it as 0

other AlternativeCost for Spells should still respect additional Cost (that should be rewritten as Static?)

Reply:
@tool4ever that should fix it?

Reply:
seems ok now